src/GoogleAnalyticsEventForwarder.js

```diff
@@ -515,7 +521,7 @@
 
                     ga('create', fieldsObject);
 
-                    setContentGroup(customFlags);
+                    applyContentGroups(customFlags);
```

Comment:
It's not clear to me why `fieldsObject` isn't necessary here.

Reply:
good call out.  technically it's not needed because fieldsobject is event level, whereas this is more global.  this code will be removed in the future. I added it even though it doesn't make a difference just for consistency